app/models/spree/komoju_gateway.rb

```diff
@@ -0,0 +1,40 @@
+module Spree
+  class KomojuGateway < Gateway
+    preference :api_key, :string
+
+    def provider_class
+      ActiveMerchant::Billing::KomojuGateway
+    end
+
+    def options
+      super.merge(login: preferred_api_key, test: preferred_test_mode)
+    end
+
+    def auto_capture?
+      false
+    end
+
+    def supports?(source)
+      true
+    end
+
+    # We need to change price from cents to dollar for Komoju gateway.
+    # Because, Komoju gateway supports JPY currency only.
+    def change_options_to_dollar(options)
+      %i(shipping tax subtotal discount).each { |key| options[key] = options[key] / 100.0 }
```

Comment:
Yes. I'll add [the link](https://github.com/spree/spree/blob/master/core/app/models/spree/payment/gateway_options.rb) to here.